pkg/services/login/authinfoservice/database/database.go

```diff
@@ -29,7 +29,9 @@ func ProvideAuthInfoStore(sqlStore db.DB, secretsService secrets.Service, userSe
 		logger:         log.New("login.authinfo.store"),
 		userService:    userService,
 	}
-	InitMetrics()
+	// FIXME: disabled the metric collection for duplicate user entries
```

Comment:
this disables prometheus metrics to be sent out, as we no longer have stats to send.